pkg/openthread/include/ot.h

```diff
@@ -110,7 +113,7 @@ void recv_pkt(otInstance *aInstance, netdev_t *dev);
  * @param[in]  dev                pointer to a netdev interface
  * @param[in]  event              just occurred netdev event
  */
-void send_pkt(otInstance *aInstance, netdev_t *dev, netdev_event_t event);
+void sent_pkt(otInstance *aInstance, netdev_t *dev, netdev_event_t event);
```

Comment:
OK. Now (in context of the API doc I get the rename). Maybe even changing it to `pkt_sent()`?

Reply:
Yes. This is called after a packet transmission finishes. So may be pkt_sent is more appropriate.